ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolConfiguration.java

```diff
@@ -64,7 +64,7 @@ enum Implementation {
   boolean DEFAULT_ENABLE_SAVE_RESTORE = false;
 
   File DEFAULT_SAVE_FILE = new File(DEFAULT_SAVE_FILE_NAME);
-  long DEFAULT_PENDING_TRANSACTIONS_LAYER_MAX_CAPACITY_BYTES = 50_000_000L;
+  long DEFAULT_PENDING_TRANSACTIONS_LAYER_MAX_CAPACITY_BYTES = 12_500_000L;
```

Comment:
Should this be 25MB or is there a non-obvious doubling going on somewhere?

Reply:
Updated the description:

> There 2 memory limited layers, ready and sparse, so to target 25MB we set the limit for a layer to 12.5MB